x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -105,10 +105,36 @@ export function AlertsPage({ routeParams }: AlertsPageProps) {
     [history, rangeFrom, rangeTo, kuery]
   );
 
+  const addToQuery = useCallback(
+    (value: string) => {
+      let output = value;
+      if (kuery !== '') {
+        output = `${kuery} and ${value}`;
+      }
+      onQueryChange({
+        dateRange: { from: rangeFrom, to: rangeTo },
+        query: output,
+      });
+    },
+    [kuery, onQueryChange, rangeFrom, rangeTo]
+  );
+
   const setRefetch = useCallback((ref) => {
     refetch.current = ref;
   }, []);
 
+  useEffect(() => {
+    if (fetcherStatus === 'success') {
+      const nextSearchParams = new URLSearchParams(history.location.search);
+
+      nextSearchParams.set('kuery', 'kibana.alert.status: "open"'); // TODO this needs to be changed, this should be part of another dependent PR that updates the Status column values to Active and recovered
+      history.push({
+        ...history.location,
+        search: nextSearchParams.toString(),
+      });
+    }
+  }, [fetcherStatus, history]);
+
```

Comment:
What's the advantage of this over just changing the default of `kuery` in line 50 to the desired filter?

```typescript
  const {
    query: {
      rangeFrom = 'now-15m',
      rangeTo = 'now',
      kuery = `${ALERT_STATUS}: "open"`,
      status = 'open',
    },
  } = routeParams;
```

Reply:
True I thought about that in the beginning, but then I thought of the scenario that user might not have any alerts. In this case it didn't make sense to me to filter for something. What do you think? The suggested code indeed is much simpler.

Reply:
No idea what `success` signals, but I'd go for the simpler and more predictable solution at this point.

Reply:
OK I pushed the simpler implementation as suggested